WordleGame/WordleGame/Extensions/UITextField + Extension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UITextField + Extension.swift
+//  WordleGame
+//
+//  Created by Alex on 4.05.22.
+//
+
+import Foundation
+import UIKit
+
+private var __maxLengths = [UITextField: Int]()
+
+extension UITextField {
+    @IBInspectable var maxLength: Int {
+        get {
+            guard let l = __maxLengths[self] else {
+                return 150 // (global default-limit. or just, Int.max)
+            }
+            return l
+        }
+        set {
+            __maxLengths[self] = newValue
+            addTarget(self, action: #selector(fix), for: .editingChanged)
+        }
+    }
+    @objc func fix(textField: UITextField) {
+        if let t = textField.text {
+            textField.text = String(t.prefix(maxLength))
+        }
+    }
+}
```

Comment:
Why do we need this fix?

Reply:
Sorry, this extension from previous idea of realisation.)